app/assets/javascripts/application.js

```diff
@@ -1,5 +1,2 @@
-//= require jquery
-//= require jquery_ujs
 //= require turbolinks
-//= require_directory ./app
-//= require_self
+//= require app/utils
```

Comment:
no great reason -- I just kind of prefer the explicitness of referring to it by name, especially since it's the only fragment loaded; re: the 'app' sub directory, I didn't want to pollute the top-level folder with anything except the application.js